### DISCUSSION TITLE
Is it possible to place cursor or use other templator commands?

### DISCUSSION THREAD

**Discussion by Tejeev** [mProjectsCode/obsidian-meta-bind-plugin#361] [Q&A]
Like adding `tp.file.cursor(1)` does for when creating file from template or adding a template.
I have a button that inserts a template when I click on it (waiting on the feature of relative line for insertion for this to really be awesome!), but it does not respect the tp command to place cursor when it's executed on the page.  
I use this as a sort of ledger.  The button inserts a line above it from a template that prints the date a `; ` and allows you to input the entry.  I'd like to place the cursor after that `; ` once the button executes but adding a `tp.file.cursor()` does nothing. I've tried giving it a higher number and hitting `ctrl+enter` to move the cursor the way you can after creating a template but that didn't work.

**Comment by mProjectsCode** [comment]
I can't really answer your question as i haven't tried to do more with templater, as I don't use it often.
